external/ag-website-shared/src/components/community/components/Events.tsx

```diff
@@ -44,7 +44,7 @@ const separateEventsByDate = (events: Event[]): { upcomingEvents: Event[]; pastE
 const EventItem = ({ event }: { event: Event }) => {
     const [darkMode] = useDarkmode();
     return (
-        <a href={event.eventPage ? event.eventPage : event.recording} target="_blank" className={styles.linkWrapper}>
+        <div target="_blank" className={styles.linkWrapper}>
```

Comment:
You don't need `target="_blank"` anymore 

```suggestion
        <div className={styles.linkWrapper}>
```

Reply:
Fixed, thanks